src/server/player_connection.cpp

```diff
@@ -33,6 +33,14 @@ int player_record::game_id() const
 	return game_ ? game_->id() : 0;
 }
 
+void player_record::next_game_id()
```

Comment:
The function name sounds like it's a getter. I'd rather use `set_next_game_id()`.